src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/components/step_index_pattern/components/status_message/status_message.js

```diff
@@ -45,22 +47,40 @@ export const StatusMessage = ({
     if (allIndices.length > 1) {
       statusMessage = (
         <span>
-          Your index pattern can match any of your <strong>{allIndices.length} indices</strong>, below.
+          <FormattedMessage
+            id="kbn.management.createIndexPattern.step.status.matchAnyLabel.matchAnyDetail"
+            defaultMessage="Your index pattern can match any of your {strongIndices}, below."
+            values={{ strongIndices: (
+              <strong>
+                <FormattedMessage
+                  id="kbn.management.createIndexPattern.step.status.matchAnyLabel.strongIndicesLabel"
+                  defaultMessage="{allIndicesLength, plural, one {# index} other {# indices}}"
```

Comment:
I think we could clean up the message here for the case when a single index exists.  "any of your 1 index" sounds pretty silly.  "any of your" should be made part of the conditional.  I would suggest the messages should be: singular "...can match your index below" plural (remains the same)